benchmarks/README.md

```diff
@@ -0,0 +1,46 @@
+# Big model inference benchmarks
+
+Running inference with Accelerate on big models.
+
+## Setup
+
+These benchmarks use the `Transformers` library:
+
+```bash
+pip install transformers
+```
```

Comment:
Do they not also need `measures_util`?

Maybe add a `pip install accelerate[bench]`?

Reply:
Forgot one file :grimacing: